src/lib/createables.ts

```diff
@@ -677,6 +678,28 @@ const Createables: Createable[] = [
 		}),
 		QPRequired: 205
 	},
+	{
+		name: 'Zamorakian hasta',
+		inputItems: resolveNameBank({
+			'Zamorakian spear': 1
+		}),
+		outputItems: resolveNameBank({
+			'Zamorakian hasta': 1
+		}),
+		fishingLevel: 55,
+		smithingLevel: 5
```

Comment:
This has a quest requirement too.

> To make or wield it, you must have completed the Fishing and Smithing parts of Barbarian Training (This requires 55 Fishing). The Smithing section requires you to finish the Tai Bwo Wannai Trio quest.

```suggestion
		smithingLevel: 5,
		QPRequired: 7
```

Reply:
This also has a GP cost of 300k.

> The Zamorakian hasta is made by taking a Zamorakian spear to Otto Godblessed, along with a payment of 300,000 coins (only 150,000 if the player has completed the Elite Kandarin Diary).